apps/nrm/index.js

```diff
@@ -158,7 +158,8 @@ module.exports = {
       ],
       locals: { showSaveAndExit: true },
       fields: ['were-they-taken-somewhere-by-their-exploiter',
-        'were-they-taken-somewhere-by-their-exploiter-journey-details'],
+        'were-they-taken-somewhere-by-their-exploiter-journey-details'
+      ],
```

Comment:
There is an extra line that has been added here. Maybe to keep it consistent with other fields the square bracket and comma could be line 161.

Reply:
Thanks I have now made changes to keep the formatting consistent with other fields where the fields are on separate lines e.g. lines 288-230. For the fields above, both fields are now on separate lines from the square brackets.

Reply:
Thanks Rhodine, for clarifying these changes. Code looks good to me.